### DISCUSSION TITLE
Error: Could not fork child process: Resource temporarily unavailable (-1). DLL rebasing may be required; see 'rebaseall / rebase --help'.<title>

### DISCUSSION THREAD

**Discussion by Ritesh1707** [community/community#58149] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Git not working properly showing 
"Error: Could not fork child process: Resource temporarily unavailable (-1).
DLL rebasing may be required; see 'rebaseall / rebase --help'."
![Screenshot 2023-06-15 191810](https://github.com/community/community/assets/134948727/3486cac9-6beb-4c7f-bf35-f03adbef92fe)

**Comment by yydspg** [comment]
i meet the same error without solution! do u fix that?

**Comment by Chenk9** [comment]
有没有大佬知道这个问题怎么解决
